lib/strategy.js

```diff
@@ -89,46 +89,57 @@ JwtStrategy.JwtVerifier = require('./verify_jwt');
  */
 JwtStrategy.prototype.authenticate = function(req, options) {
```

Comment:
Optionally, this could be an async function()

JwtStrategy.prototype.authenticate = async function(req, options) { 
    // async 
    const token = await self._jwtFromRequest(req);
...
}